.ci/setup.sh

```diff
@@ -22,21 +22,27 @@ if [[ $OS_NAME == "macos" ]]; then
         wget -O conda.sh https://repo.continuum.io/miniconda/Miniconda${PYTHON_VERSION:0:1}-latest-MacOSX-x86_64.sh
     fi
 else  # Linux
-    if [[ $AZURE == "true" ]] && [[ $COMPILER == "clang" ]]; then
-        update-alternatives --install /usr/bin/clang++ clang++ /usr/bin/clang++-6.0 100
-        update-alternatives --install /usr/bin/clang clang /usr/bin/clang-6.0 100
+    if [[ $AZURE == "true" ]]; then
         sudo apt-get update
```

Comment:
may be we should update for travis as well, since line 48 will install ocl-icd-opencl-dev

Reply:
I tested without `sudo apt-get update` at Travis - worked well. But if it's not significantly enlarge the test time, lets leave this.